plugin/othttp/handler.go

```diff
@@ -168,8 +177,13 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 	opts := append([]trace.StartOption{}, h.spanStartOptions...) // start with the configured options
 
+	operation := h.operation
+	if h.operationNamer != nil {
+		operation = h.operationNamer(r)
+	}
```

Comment:
I have a slight preference for the approach taken in #617 as it doesn't depend on any conditionals.  It results in the consistent application of a span formatter, even if that is the default.

Reply:
hah that's my co-worker :) looks like he beat me to it. I'll go ahead and close this PR out in favor of his. Thanks!